data_structures/src/chain.rs

```diff
@@ -867,13 +867,15 @@ pub struct RADRetrieve {
     crate = "crate"
 )]
 pub struct RADAggregate {
-    pub script: Vec<u8>,
+    pub filters: Vec<u8>,
+    pub reducer: u32,
```

Comment:
This cannot be a `u8` because of some limitations of the `ProtobufConvert` crate.

It could be `u16`, `i16` or `i8` here, and `uint32` or `int32` in `witnet.proto`. Protobuf uses variable length encoding, so using a `uint32` is fine for our use case.

Or we could update the `ProtobufConvert` crate. With the newest features it should be possible to implement a conversion to `u8` with an attribute.